txpipe/data_types/types.py

```diff
@@ -1,4 +1,5 @@
-from .base import FitsFile, HDFFile, DataFile, YamlFile, TextFile, Directory, PNGFile
+from .base import FitsFile, HDFFile, DataFile, YamlFile
+from .base import TextFile, Directory, PNGFile, FileCollection
```

Comment:
not a big deal, but FileCollection is not used here (I was trying to look at how it gets used)

Reply:
I've tidied up the whole thing.